components/StyledSelect.js

```diff
@@ -139,6 +140,7 @@ export const makeStyledSelect = SelectComponent => styled(SelectComponent).attrs
     noOptionsMessage: () => intl.formatMessage(Messages.noOptions),
     components: getComponents(components, useSearchIcon),
     instanceId: instanceId ? instanceId : inputId,
+    theme: selectTheme,
```

Comment:
Added this one cause `theme` is what's used to customize the height with `react-select`, but we can't accept a `theme` prop directly as `styled-components` already sets one.